cpp/turbodbc_arrow/Library/src/python_bindings.cpp

```diff
@@ -20,5 +21,6 @@ PYBIND11_MODULE(turbodbc_arrow_support, module)
     pybind11::class_<arrow_result_set>(module, "ArrowResultSet")
         .def("fetch_all", &arrow_result_set::fetch_all);
 
-    module.def("make_arrow_result_set", make_arrow_result_set);
+    module.def("make_arrow_result_set", make_arrow_result_set,
+        pybind11::arg("result_set"), pybind11::arg("strings_as_dictionary") = false);
```

Comment:
The default is set at the Python level in `cursor.py`. There is no need to specify the default a second time here, as it burdens to code with additional pybind11 options and introduces the risk of diverging defaults.